src/utility/time/TimeService.cpp

```diff
@@ -51,7 +51,7 @@ static time_t const EPOCH_AT_COMPILE_TIME = cvt_time(__DATE__);
 
 TimeService::TimeService()
 : _con_hdl(nullptr)
-#ifdef ARDUINO_ARCH_SAMD
+#if defined (ARDUINO_ARCH_SAMD) || defined (TARGET_PORTENTA_H7)
```

Comment:
Just curious: What's the `TARGET_PORTENTA_H7` define about?

Reply:
since RTC is not supported yet on the NANO 2040 i had to restrict the changes only to the PORTENTA. 
TARGET_PORTENTA_H7 is defined by the mbed build system during the build of libmbed.a

https://github.com/arduino/ArduinoCore-mbed/blob/7de71b76db843d0cc5b0d066c276ec295c7993d4/variants/PORTENTA_H7_M7/defines.txt#L68

Reply:
Thank you :+1: